JSDemos/images/icons/status-1.svg

```diff
@@ -0,0 +1 @@
+<svg id="Слой_1" data-name="Слой 1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><title>status-1</title><path d="M8,2A6,6,0,1,1,2,8,6,6,0,0,1,8,2M8,0a8,8,0,1,0,8,8A8,8,0,0,0,8,0Z" style="fill:#b1b1b1"/><polygon points="9 4 7 4 7 9 11 9 11 7 9 7 9 4 9 4" style="fill:#b1b1b1"/></svg>
```

Comment:
We should not use russian words in our demos. Take a look at **id** and **data-name** attribures.